lib/client/api.go

```diff
@@ -2066,6 +2070,18 @@ func PlayFile(ctx context.Context, tarFile io.Reader, sid string) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	// Checks to see if attempting to interactively play desktops or not SSH, Kube types
+	if len(sessionEvents) > 0 {
+		if sessionEvents[0].GetType() == events.WindowsDesktopSessionStartEvent {
```

Comment:
I'd convert this to a switch:

```
switch typ := sessionEvents[0].GetType(); typ {
case events.WindowsDesktopSessionStartEvent:
    return trace.BadParameter(...)
case events.AppSessionStartEvent, events.DatabaseSessionStart:
    return trace.BadParameter(...)
case events.SessionStart:
    // proceed without error
default:
    return trace.BadParameter("unknown session type %q", typ)
}
```

This makes it a bit more explicit as opposed to relying on "anything other than session start"

(I'd probably do the same in up on line 1991 too)

Reply:
Thanks @zmb3 , updated to use switch.